bigquery/google/cloud/bigquery/client.py

```diff
@@ -1744,26 +1741,28 @@ def _initiate_resumable_upload(self, stream, metadata, num_retries):
     def _do_multipart_upload(self, stream, metadata, size, num_retries):
         """Perform a multipart upload.
 
-        :type stream: IO[bytes]
-        :param stream: A bytes IO object open for reading.
+        Args:
+            stream (IO[bytes]): A bytes IO object open for reading.
+
+            metadata (Dict): The metadata associated with the upload.
 
-        :type metadata: dict
-        :param metadata: The metadata associated with the upload.
+            size (int):
+                The number of bytes to be uploaded (which will be read
+                from ``stream``). If not provided, the upload will be
+                concluded once ``stream`` is exhausted (or :data:`None`).
 
-        :type size: int
-        :param size: The number of bytes to be uploaded (which will be read
-                     from ``stream``). If not provided, the upload will be
-                     concluded once ``stream`` is exhausted (or :data:`None`).
+            num_retries (int):
+                Number of upload retries. (Deprecated: This
+                argument will be removed in a future release.)
 
-        :type num_retries: int
-        :param num_retries: Number of upload retries. (Deprecated: This
-                            argument will be removed in a future release.)
+        Returns:
+            (requests.Response):
+                The "200 OK" response object returned after the multipart
+                upload request.
 
-        :rtype: :class:`~requests.Response`
-        :returns: The "200 OK" response object returned after the multipart
-                  upload request.
-        :raises: :exc:`ValueError` if the ``stream`` has fewer than ``size``
-                 bytes remaining.
+        Raises:
+            :exc:`ValueError` if the ``stream`` has fewer than ``size``
```

Comment:
This `:exc:` marking should be deleted too? Didn't found anything about it in https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html#example-google

Reply:
Yes, exception types are formatted similarly to return types. I'd suggest the following formatting:

```suggestion
            ValueError:
                If the ``stream`` has fewer than ``size``
```